#include <iostream>
#include<conio.h>

using namespace std;

int main() {
   int var = 0;

   cout << "Simple C++ Program for Basic Exception Handling\n";
   // try block - for exception code
   try {
      // Inside try block
      if (var == 0) {
         throw var;
         // Skip all code after throw
      }
   }   // catch block
   catch (int ex) {
      // Code executed when exception Catch
      cout << "Exception Catch : Value :" << ex;
   }

   getch();
   return 0;
}
